locales/pt_BR/json.json

```diff
@@ -995,7 +995,7 @@
     "View": "Visualizar",
     "View Invitations": "Ver convites",
     "View Receipt": "Veja o recibo",
-    "Virgin Islands": "Virgin Islands",
+    "Virgin Islands": "Ilhas Virgins",
```

Comment:
In Brazilian Portuguese, the correct is `Ilhas Virgens` not `Ilhas Virgins`

Reply:
sorry, it went unnoticed